drift/example/main.dart

```diff
@@ -5,13 +5,54 @@ import 'package:drift/native.dart';
 
 part 'main.g.dart';
 
+class TodoCategories extends Table {
+  IntColumn get id => integer().autoIncrement()();
+  TextColumn get name => text()();
+}
+
 class TodoItems extends Table {
   IntColumn get id => integer().autoIncrement()();
   TextColumn get title => text()();
   TextColumn get content => text().nullable()();
+  IntColumn get categoryId => integer().references(TodoCategories, #id)();
+}
+
+class TodoCategoryItemCount extends View {
+  late $TodoItemsTable todoItems;
+  late TodoCategories todoCategories;
```

Comment:
I'm not entirely sold on the `late` syntax here.

Could we make it so that a user makes a view class abstract, writes `TodoCategories get todoCategories;` and lets drift generate an implementation of the view class (which would then inherit `ViewInfo`) that overrides the getters.

Reply:
Yeah. That's a good point.